jenkins-pipelines/manager/debian9-manager-sanity.jenkinsfile

```diff
@@ -13,6 +13,7 @@ managerPipeline(
     test_config: 'test-cases/manager/manager-regression-multiDC-set-distro.yaml',
     ami_id_monitor: 'ami-0cfac3931b2a799d1', // Debian stretch image
     ami_monitor_user: 'admin',
+    aws_root_disk_name_monitor: '/dev/xvda',
```

Comment:
@bentsi asked to remove it from the pipeline, and put it in the yaml file

Reply:
We discussed about this, and decided that for now this will be a proper fix, since there is no way to change it from the job params